vmdb/app/helpers/container_service_helper/textual_summary.rb

```diff
@@ -21,6 +21,7 @@ def textual_group_port_configs
 
   def textual_group_relationships
     items = %w(ems container_groups)
+    items = %w(ems container_routes)
```

Comment:
This is replacing the `items` value. You should change the line in:

    items = %w(ems container_routes container_groups)